chore(docs): fix Custom Client Session Handling example (#4310)
nextauthjs/next-auth: docs/docs/getting-started/client.md

}

function Auth({ children }) {

>   // with required:true status can only be loading or authenticated

Reviewer: ```suggestion
  // if `{ required: true }` is supplied, `status` can only be "loading" or "authenticated"
```
